### PR TITLE
[back] feat: added video datetime in metadata

### PR DESCRIPTION
**Pull request by tbrnd**
PR addressing #818

### CODE REVIEW

**Comment by GresilleSiffle** [comment]
Thank you @tbrnd 

It looks good to me. I'll just update a little bit the front end before merging, so that the publication date are displayed in a more human friendly way.

capture2

**Comment by amatissart** [comment]
What about existing data? Do we need to add a migration or the new serializer is compatible with the date-only format?

**Comment by GresilleSiffle** [comment]
> What about existing data? Do we need to add a migration or the new serializer is compatible with the date-only format?

The new serializer is compatible with the date-only format, but will consider all videos as published at midnight UTC. I think adding a migration could be good to avoid having different kind of data in the database.

**Comment by GresilleSiffle** [comment]
It looks like I broke the e2e tests with the new date display. I'll fix them.

File: backend/tournesol/tests/test_api_video.py
Comment (tbrnd) [inline backend/tournesol/tests/test_api_video.py:316]: realized this this morning but this test should fail, 3 is expected (19:44:24 < 19:45:00), please do not merge before this is fixed

**Review by GresilleSiffle — APPROVED** [review]

**Pull request merged**
